### PR TITLE
Allows admins to edit transform matrices.

### PR DESCRIPTION
**Pull request by PJB3005**
https://dl.dropboxusercontent.com/u/107028026/ShareX/2016/07/2016-07-28_22-59-13.webm

### CODE REVIEW

**Comment by ComicIronic** [comment]
The VV window is the only good thing oldcoders ever did

**Comment by PJB3005** [comment]
Code wise is it's pure cancer.

**Comment by clusterfack** [comment]
VV is so bizarrely out of step with everything else I wonder whose library they stole it from

**Comment by ghost** [comment]
have a reset button, please.

**Comment by RemieRichards** [comment]
Yeah add a setting for an identity matrix, and a setting to input a list (DM can make matrices from lists) 

> It is also legal to assign a list with six values, which is equivalent to using a matrix.

**Comment by 9600bauds** [comment]
just like self merge when this is ready lol

**Comment by ghost** [comment]
needs a reset button aaaaaaaaaaa

**Comment by clusterfack** [comment]
@PJB3005 Please

File: code/modules/admin/verbs/massmodvar.dm [hunk 1]
```diff
@@ -441,5 +445,48 @@
 						if (A.type == O.type)
 							A.vars[variable] = O.vars[variable]
 
+		if ("matrix")
+			var/matrix/new_value = modify_matrix_menu(O.vars[variable], verbose = FALSE)
+			if (!new_value)
+				return
+
+			O.vars[variable] = new_value
```
Comment (ComicIronic) [inline code/modules/admin/verbs/massmodvar.dm:453]: Tell me this block isn't copypasted, over and over and over.
Tell me it ain't so.

Comment (PJB3005) [inline code/modules/admin/verbs/massmodvar.dm:453]: It is.

Literally everything about VV is a huge copy pasted mess.

**Comment by PJB3005** [comment]
Alright done.

File: code/modules/admin/verbs/modifyvariables.dm [hunk 2]
```diff
@@ -605,6 +605,22 @@ var/list/forbidden_varedit_object_types = list(
...
+
+		if ("Manual")
+			var/list/numbers = splittext(input("Enter the matrix components as a comma separated list.") as text|null, ",")
+			if (!numbers || numbers.len != 6)
+				to_chat(src, "Cancelled or not enough arguments provided.")
+
+			else
+				var/list/newnumbers = list()
+				for (var/number in numbers)
+					number = text2num(number)
```
Comment (ComicIronic) [inline code/modules/admin/verbs/modifyvariables.dm]: What happens if you text2num a non-number?

Comment (PJB3005) [inline code/modules/admin/verbs/modifyvariables.dm]: Results in `null`, alright I'll fix it.

**Pull request merged**